packages/ChangesReloaded-Tests.package/CRTest.class/instance/tearDown.st

```diff
@@ -0,0 +1,4 @@
+running
+tearDown
+
+	self changesBrowser: nil
```

Comment:
As discussed in person, this doesn't do anything.

Also, `setUp` calls `super setUp`. Do we also need to call `super tearDown`? (I'm not sure about this one and, of course, this only applies if we would need this method at all.)